charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -145,13 +217,53 @@ groups:
 
   # Recording rules for the sum of vpa recommendations for all the control-planes
   - record: seed:vpa_status_recommendation_target:sum
-    expr: sum(vpa_status_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"})
+    expr: sum(vpa_status_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"}) by (resource)
   
   # Recording rules for the sum of hvpa applied recommendations for all the control-planes
   - record: seed:hvpa_status_applied_vpa_recommendation_target:sum
-    expr: sum(hvpa_status_applied_vpa_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"})
+    expr: sum(hvpa_status_applied_vpa_recommendation{recommendation="target", namespace=~"((shoot-|shoot--)(\\w.+))"}) by (resource)
 
   # Recording rules for pod count for all the control-planes
   - record: seed:kube_pod_info:count_cp
     expr: count(kube_pod_info{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for pod count by status for all the control-planes
+  - record: seed:kube_pod_status_ready:sum_cp_by_condition
+    expr: sum(kube_pod_status_ready{namespace=~"((shoot-|shoot--)(\\w.+))"}) by (condition)
+
+  # Recording rules for pod count by phase for all the control-planes
+  - record: seed:kube_pod_status_phase:sum_cp_by_phase
+    expr: sum(kube_pod_status_phase{namespace=~"((shoot-|shoot--)(\\w.+))"}) by (phase)
 
+  # Recording rules for container count for all the control-planes
+  - record: seed:kube_pod_container_info:count_cp
+    expr: count(kube_pod_container_info{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for container restart count for all the control-planes
+  - record: seed:kube_pod_container_status_restarts_total:sum_cp
+    expr: sum(kube_pod_container_status_restarts_total{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for deployment spec replicas for all the control-planes
+  - record: seed:kube_deployment_spec_replicas:sum_cp
+    expr: sum(kube_deployment_spec_replicas{namespace=~"((shoot-|shoot--)(\\w.+))"})
+  
+  # Recording rules for deployment status replicas for all the control-planes
+  - record: seed:kube_deployment_status_replicas:sum_cp
+    expr: sum(kube_deployment_status_replicas{namespace=~"((shoot-|shoot--)(\\w.+))"})
```

Comment:
All *_cp metrics need to aggregate by (namespace) otherwise there is no label for each namespace.

Reply:
Same as [above](https://github.com/gardener/gardener/pull/1958#discussion_r384263240).